prow/plugins/plugins.go

```diff
@@ -251,7 +254,7 @@ func NewFakeConfigAgent() ConfigAgent {
 // the file can't be read or the configuration is invalid.
 // If checkUnknownPlugins is true, unrecognized plugin names will make config
 // loading fail.
-func (pa *ConfigAgent) Load(path string, checkUnknownPlugins bool) error {
+func (pa *ConfigAgent) Load(path string, supplementalPluginConfigDirs []string, supplementalPluginConfigFileSuffix string, checkUnknownPlugins bool) error {
```

Comment:
This looks to me that the config will be something like:
```
--supplementalPluginConfigDirs=dir_a
--supplementalPluginConfigDirs=dir_b
--supplementalPluginConfigFileSuffix=custom.yaml 
```

This should work majority of cases, is there any chance in the future that the suffix might differ between directories? If so might want to consider changing this to a map instead

Reply:
I think this is easy to change, should we actually end up having that use-case